test/PizzeriaTest.cpp

```diff
@@ -65,3 +63,23 @@ TEST_F(PizzeriaTest, calculatePriceForPizzaMock)
 
     delete mock;
 }
+
+TEST_F(PizzeriaTest, orderStubAndTwoMockPizzas) {
+    // Given
+    StrictMock<PizzaMock>* strict_mock = new StrictMock<PizzaMock>{"strict", 1, minutes(0)};
+    NiceMock<PizzaMock>* nice_mock = new NiceMock<PizzaMock>{"nice", 2, minutes(3)};
+    Pizzas pizzas = {new PizzaStub{"STUB"}, strict_mock, nice_mock};
+    EXPECT_CALL(tm, sleep_for(_)).Times(3);
+    EXPECT_CALL(*strict_mock, getPrice());
+    EXPECT_CALL(*strict_mock, getName());
+    EXPECT_CALL(*strict_mock, getBakingTime());
+
+    // When
+    auto orderId = pizzeria.makeOrder(pizzas);
+    auto price = pizzeria.calculatePrice(orderId);
```

Comment:
what about that price? It would be nice to check if it is equal to expected price.